whitelist/whitelist.go

```diff
@@ -0,0 +1,73 @@
+// Package whitelist provides methods for managing an app's whitelisted IP's.
+package whitelist
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	deis "github.com/deis/controller-sdk-go"
+	"github.com/deis/controller-sdk-go/api"
+)
+
+// List IP's whitelisted for an app.
+func List(c *deis.Client, appID string) (api.Whitelist, error) {
```

Comment:
My concern is having the same information in two different places. We could also remove it from app settings if you prefer (it might be confusing to have get/set but no list).